src/main/java/javaserver/Responders/FormResponder.java

```diff
@@ -0,0 +1,45 @@
+package javaserver.Responders;
+
+import javaserver.Form;
+import javaserver.HTTPStatusCode;
+import javaserver.Request;
+import javaserver.Response;
+
+public class FormResponder implements Responder {
+
+	private String[] supportedMethods;
+	private Form form;
+
+	public FormResponder(String[] supportedMethods, Form form) {
+		this.supportedMethods = supportedMethods;
+		this.form = form;
+	}
+
+	@Override
+	public Response getResponse(Request request) {
+		return new Response.ResponseBuilder(getStatusLine(request))
+      .body(getData(request))
```

Comment:
I believe I'm using tabs. Here are the settings in my vimrc
![screen shot 2016-04-07 at 10 36 26 am](https://cloud.githubusercontent.com/assets/10511338/14360748/a67cd1c0-fcac-11e5-9636-bce21d906413.png)